.eslintrc.js

```diff
@@ -20,13 +20,16 @@ module.exports = {
         'arrow-body-style': ['warn', 'as-needed'],
         camelcase: 'off',
         'class-methods-use-this': 'off',
+        'consistent-return': 0,
```

Comment:
As of this morning on Master there are 1864 problems:

![Screenshot 2020-04-06 at 12 59 16](https://user-images.githubusercontent.com/864612/78551767-8b465280-7806-11ea-8cdd-fedac30a830a.png)

The rule changes i've made were quite small and basically because the code is unreadable in places cause of the error highlighting etc.

RNV styling should be in line with our other projects also